README.rst

```diff
@@ -21,60 +21,64 @@ in a folder. Currently it only supports Microsoft Windows, GNOME (and
 its many derivatives), and macOS. Contributions to support other OSs/desktop
 environments are highly welcome.
 
-`WinPython <http://winpython.github.io>`__ and `Anaconda <https://www.anaconda.com/distribution>`__/`Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__ distributions are supported. If run from a conda environment other than `root`, the name of the environment will be specified in brackets in the context menu name. 
+`WinPython <http://winpython.github.io>`__ and `Anaconda <https://www.anaconda.com/distribution>`__/`Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__ distributions are supported. If run from a conda environment other than `root`, the name of the environment will be specified in brackets in the context menu name.
 
 Microsoft Windows
 ~~~~~~~~~~~~~~~~~
 
-.. figure:: images/jupyter_cm_windows.png
+.. figure:: https://github.com/hyperspy/start_jupyter_cm/raw/master/images/jupyter_cm_windows.png
    :alt: Jupyter context menu entries in windows
 
    Jupyter context menu entries in windows.
 
-In addition to starting the QtConsole, the Jupyter Notebook or the Jupyter Lab, 
-and launching the default browser, in Microsoft Windows the process runs from 
+In addition to starting the QtConsole, the Jupyter Notebook or the Jupyter Lab,
+and launching the default browser, in Microsoft Windows the process runs from
 a terminal. Closing the terminal closes the QtConsole or the Jupyter server.
 Single and all users installations are supported, see installation instructions below.
 
-Linux (Nautilus and Caja)
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Linux
+~~~~~
+
+On linux, the supported file managers are: Nautilus, Caja and Dolphin. With
+Nautilus and Caja, the shortcut will appear in the *Scripts* menu and with
+Dolphin, it will appear in the *Actions* menu.
 
-.. figure:: images/jupyter_cm_gnome.png
-   :alt: Jupyter context menu entries in gnome
+.. figure:: https://github.com/hyperspy/start_jupyter_cm/raw/master/images/jupyter_cm_gnome.png
+   :alt: Jupyter context menu entries in Nautilus
+   :width: 400px
```

Comment:
The size is ignored on github and this seems to be known bug: https://github.com/github/markup/issues/295, maybe worth using html...